docs/arch.md

```diff
@@ -23,7 +23,9 @@ To simplify and structure the development, the API is divided into a few microse
 | EO Data Discovery          | Describes which datasets and image collections are available at the back-end. |
 | Process Discovery          | Provides services to find out which processes a back-end provides, i.e., what users can do with the available data. |
 | UDF Runtime Discovery      | Allows discovering the programming languages and their runtime environments to execute user-defined functions. |
-| Job Management             | Organizes and manages jobs that run processes on back-ends.  |
-| Result Access and Services | Services to download data and job results, e.g. as WCS or WMTS service. |
-| User Data Management       | Manage user content and accounting. Might be split into multiple microservices, e.g. User Files and User Process Graphs might be separated. |
-| Authentication             | Authentication of users.                                     |
+| Job Management             | Organizes and manages jobs that run processes on back-ends. |
+| File Management            | Organizes and manages user-uploaded files. |
+| Process Graph Management   | Organizes and manages user-defined process graphs.  |
+| Web Service Management     | Web services to access data and job results, e.g. as WCS or WMTS service. |
```

Comment:
@mkadunc The name obviously originates from OGC "web services", e.g. WMS, WCS etc. I am open to introduce a better name, but in my opinion "Result Service Management" doesn't fully reflect the current grouping of the API endpoints as the download of job results (/jobs/:job_id/results) is not included here and only "external" services are bundled in this "category" (or microservice).

Reply:
When I see "web services" I think of REST, SOAP, OGC. When a web service has a section called web service management, I'd expect it to handle configuration and management of the service itself, not deal with subordinate services...

I see your point, and I agree that results might not be a good name. If we focus on these services being non-openEO, we could call them auxiliary / ancillary / foreign

We could focus on the services being "spawned" by the back-end, and call them subordinate / supplementary / secondary / collateral ...

We could also focus on the services' intent (usually the best way to approach naming in SW development). It seems that they provide asynchronous, on-demand access to results and call these  Async-Access-Service Management (but, if this will not be the only means of async access, this name might not be appropriate).

Reply:
Okay, I get your point about the web services, but the services are usually REST/OGC/SOAP-based, too. These secondary services are usually configured and spawned through the openEO API. I don't think they are necessarily always asynchronous and may even be not just for accessing the processed data. So maybe something like "[Foreign|Secondary|External] [Web|Access] Services Management" to make clear we are not managing the openEO API itself?

Reply:
Glad we're converging, and thanks for indulging me :)

I'd go with "Secondary Services Management" from your options above - it's generic enough that it can be used for not just processed data, and it's descriptive enough to make it clear that these are not the core openEO services...

Reply:
Okay, great. I changed that.